### ISSUE TITLE
Telemetry: Collect actualy OS, not only type

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#5199]
Currently we only collect the OS type, not actually the distribution.

Looking at the admin page on /admin/reports/telemetry - I'd assume that we need something other than just `PHP_OS` to get what we want. Otherwise it's either Linux or Windows, very likely "Linux" in 99% - which is pointless.

![telemetry-os](https://user-images.githubusercontent.com/842066/132121856-38944639-f787-4778-95d2-6835222c751c.png)

Options:

- php_uname()
- getenv()
- phpinfo()

... reduced to the info we need.

**Comment by indigoxela** [comment]
Off topic: Webserver and sapi would be more valuable information. For instance, we had discussions about issues, that fixed something for mod_php, and there were concerns that mod_php is rarely used in real hosting environments.

- `getenv('SERVER_SOFTWARE')` Apache or other
- `php_sapi_name();` Returns the type of interface between web server and PHP (fcgi / fpm / ...)

Not sure, if anyone agrees. :wink:

**Comment by klonos** [comment]
I agree with collecting `php_sapi_name()` and webserver, but that's different to what the title of this issue says (unless we want to handle OS as well as all these things here).

I actually also looked at that "Linux" thing and thought that we should be getting more info than just that. I started doing some light research, but then stopped when I read this:

https://www.php.net/manual/en/function.php-uname.php
>On some older UNIX platforms, it may not be able to determine the current OS information in which case it will revert to displaying the OS PHP was built on. This will only happen if your uname() library call either doesn't exist or doesn't work.
